Privacy.md

```diff
@@ -0,0 +1,85 @@
+# Fruit Radar Privacy Policy.
+
+This privacy policy document is to clearly state and provides information on what, how and why personal data or information is collected, used and disclosed when this software is utilized and the choices you have associated with that data.
+
+The objective of this policy document is to provide an open environment between developer and users of this software by openly stating what information the developer collects or not. By utilizing this software, it means you accept all the terms and conditions of the developer peculiar to only Fruit Radar software. This Policy applies to this software and the developer would not be held responsible for the content or privacy practices on any third party app not operated by the developer to which this software links or that links to this software, therefore it is advisable to read up the privacy policies of third party services that may be linked to this software. <hr>
+
```

Comment:
A note: In Markdown, you can use `***`

***

to make a hr

Reply:
Oh okay. Thanks for letting me know.